addons/csw/functions/fnc_aceRearmGetCarryMagazines.sqf

```diff
@@ -20,7 +20,7 @@ params ["_vehicle", ["_targetTurret", true, [[], true]]];
 
 if (!(_vehicle isKindOf "StaticWeapon")) exitWith { [[],[]] }; // limit to statics for now
 // Assembly mode: [0=disabled, 1=enabled, 2=enabled&unload, 3=default]
-if ((GVAR(ammoHandling) == 0) && {!([false, true, true, GVAR(defaultAssemblyMode)] select (_vehicle getVariable [QGVAR(assemblyMode), 3]))}) exitWith { [[],[]] };
+if (!([false, true, true, GVAR(defaultAssemblyMode)] select (_vehicle getVariable [QGVAR(assemblyMode), 3]))) exitWith { [[],[]] };
```

Comment:
I'd say remove these lines and just enable it all the time regardless of defaultAssemblyMode
otherwise we are removing functionality that previously existed

Reply:
I'd still like an option for explicitly disabling CSW (assemblyMode 0 handles this, and is relevant for #9238). Maybe just adding a check for `(_target getVariable [QGVAR(assemblyMode), 3]) == 0` instead?